src/Hosting/TestHost/ref/Microsoft.AspNetCore.TestHost.netcoreapp3.0.cs

```diff
@@ -28,12 +29,14 @@ public partial class TestServer : Microsoft.AspNetCore.Hosting.Server.IServer, S
         public TestServer() { }
         public TestServer(Microsoft.AspNetCore.Hosting.IWebHostBuilder builder) { }
         public TestServer(Microsoft.AspNetCore.Hosting.IWebHostBuilder builder, Microsoft.AspNetCore.Http.Features.IFeatureCollection featureCollection) { }
-        public TestServer(Microsoft.AspNetCore.Http.Features.IFeatureCollection featureCollection) { }
```

Comment:
We need to keep all of the old constructors for back compat. Add this overload back.

Reply:
Nevermind, this one doesn't need to come back, it wasn't present in 2.2 either.